event-queue/README.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: pattern
+title: Event Queue
+folder: event-queue
+permalink: /patterns/event-queue/
+categories: Concurrency
+tags:
+ - Java
+ - Difficulty Intermediate
+ - Queue
+---
+
+## Intent
+Event Queue is a good pattern if You have a limited accesibility resource (for example: 
+Audio or Database), but You need to handle all the requests that want to use that.
+It puts all the requests in a queue and process them asynchronously.
+Gives the resource for the event when it is the next in the queue and in same time
+removes it from the queue.
```

Comment:
Sorry about that, but my english is not the best :( Please help me what is wrong. What do I have to change?